### PR TITLE
Modified Registration.html

### LINKED ISSUE #2: As a User, I want to be able to register to the platform, so I can start exchanging vinyls
Implement a user registration page according to the following design:
image

### PR DESCRIPTION
**Pull request by nyitrai87**
I modified the Registration.html as you requested, to fit to the template.

### CODE REVIEW

**Comment by nyitrai87** [comment]
Since then I fixed the gender choose. Now you can choose only one of the 2 options in the same time.

File: src/registration.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    </label>
+                </div>
+                <div>
+                    <label>
+                        Last Name <input id="lastname" type="text"/>
+                    </label>
+                </div>
+                <div>
+                    <label>
+                        Male <input type="radio" name="gender" id="male" value="0">
```
Comment (attilagyongyosi) [inline src/registration.html:24]: 👍 perfect use of the `name` attribute to group radio buttons together. Nice!

File: src/registration.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    </label>
+                </div>
+                <div>
+                    <label>
+                        Date of Birth <input id="dateofbirth" type="date"/>
+                    </label>
+                </div>
+                <div>
+                    <label>
+                        Email Address <input id="email" type="text"/>
```
Comment (attilagyongyosi) [inline src/registration.html]: There is a specific `type` value for email fields, surprisingly called `email`. With that, your browser will enforce a typical email address format for that input field.
Check out this documentation on the various types of... `type`s: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

- [x] Change type attribute to `email`.

**Review by attilagyongyosi — CHANGES_REQUESTED** [review]

File: src/registration.html (hunk 2 above)
Comment (nyitrai87) [reply src/registration.html]: Done, text type changed to email. :)

**Review by attilagyongyosi — APPROVED** [review]
Changes have been approved.

**Pull request merged**